### PR TITLE
[ADD/IMP] outplacement_order_interpreter: Fixed issue of Interpreter …

### PR DESCRIPTION
**Pull request by rupareliyahemangi145**
…booking status and Added logger info when user click on cancel button of booking cancellation popup.

### CODE REVIEW

**Comment by rupareliyahemangi145** [comment]
Hello @S-Arnshed-Capgemini 

Can you update this code on production manually instead of new building?
Thanks.

Regards,
Hemangi.

File: outplacement_order_interpreter/models/mail_activity.py [hunk 1]
```diff
@@ -748,18 +748,14 @@ def interpreter_cancel_booking(self):
...
-            'res_id': self.res_id,
-            'model': self.res_model,
-        })
-        _logger.info(f'{author} {message_id.body}')
-        self.active = False
+        link = "mailto:team-crm@arbetsformedlingen.se?subject=Avboka tolk med referens " + \
+               str(self.interpreter_booking_ref) + " och KA-Nr: " + str(self.interpreter_ka_nr) + "."
+        message = "Skicka ett epostmeddelande till "
+        email_address = 'team-crm@arbetsformedlingen.se'
+        message += '<a href=' + link + '>' + email_address + '</a>'
```
Comment (SebastianGieser-Vertel) [inline outplacement_order_interpreter/models/mail_activity.py]: Please use proper string building instead, e.g. message = f"Skicka ett epostmeddelande till <a href={link}> {email_address} </a>"

**Review by SebastianGieser-Vertel — CHANGES_REQUESTED** [review]
Please update the manifest file

**Comment by fredrikarvas-vertel** [comment]
@SebastianGieser-Vertel - may I ask you to fix these issues since Hemangi is not working more today (she is 3,5 hours ahead of us and starts earlier).

**Comment by rupareliyahemangi145** [comment]
Hello @FredrikArvas 

I am working and will commit in next 5 mins.

**Comment by rupareliyahemangi145** [comment]
@FredrikArvas  and @SebastianGieser-Vertel  I've updated code. Please check it and let me know if anything.

File: outplacement_order_interpreter/models/mail_activity.py [hunk 2]
```diff
@@ -752,9 +752,8 @@ def interpreter_cancel_booking(self):
                str(self.interpreter_booking_ref) + " och KA-Nr: " + str(self.interpreter_ka_nr) + "."
         message = "Skicka ett epostmeddelande till "
         email_address = 'team-crm@arbetsformedlingen.se'
-        message += '<a href=' + link + '>' + email_address + '</a>'
-        message += " och ange referensnummer " \
-            + str(self.interpreter_booking_ref) + ".Aktuellt KA-Nr: " + str(self.interpreter_ka_nr)
+        message += f"<a href='{link}'>{email_address}</a>"
```
Comment (SebastianGieser-Vertel) [inline outplacement_order_interpreter/models/mail_activity.py]: still using +=, use only =

File: outplacement_order_interpreter/models/mail_activity.py [hunk 3]
```diff
@@ -752,9 +752,8 @@ def interpreter_cancel_booking(self):
...
+        message += f" och ange referensnummer {self.interpreter_booking_ref}. Aktuellt KA-Nr: {self.interpreter_ka_nr}"
```
Comment (SebastianGieser-Vertel) [inline outplacement_order_interpreter/models/mail_activity.py]: add to above row

**Review by SebastianGieser-Vertel — CHANGES_REQUESTED** [review]

**Pull request merged**